crossbeam-deque/src/deque.rs

```diff
@@ -284,6 +284,28 @@ impl<T> Worker<T> {
         }
     }
 
+    /// Checks if the worker and the provided stealer are pointing to the same underlying queue.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use crossbeam_deque::Worker;
+    ///
+    /// let w_1 = Worker::<i32>::new_lifo();
+    /// let w_2 = Worker::<i32>::new_fifo();
+    ///
+    /// let s_1 = w_1.stealer();
+    /// let s_2 = w_2.stealer();
+    ///
+    /// assert!(w_1.is_same_as(&s_1));
+    /// assert!(w_2.is_same_as(&s_2));
+    /// assert!(!w_1.is_same_as(&s_2));
+    /// assert!(!w_2.is_same_as(&s_1));
+    /// ```
+    pub fn is_same_as(&self, stealer: &Stealer<T>) -> bool {
```

Comment:
I don't think this method name is easy to understand, but I'm not sure what the good name would be.

Reply:
Not sure if `Worker::ptr_eq` is better. I can't seem to come up with a better name here either.

Reply:
How about `Worker::donates_to`?